back/engines/commercial/idea_custom_fields/app/controllers/idea_custom_fields/web_api/v1/admin/idea_custom_fields_controller.rb

```diff
@@ -82,15 +82,6 @@ def update_all
       render json: { errors: e.errors }, status: :unprocessable_entity
     end
 
-    def custom_form
-      authorize CustomField.new(resource: @custom_form), :index?, policy_class: IdeaCustomFieldPolicy
-      render json: ::WebApi::V1::CustomFormSerializer.new(
-        @custom_form,
-        params: jsonapi_serializer_params,
-        include: []
-      ).serializable_hash
-    end
```

Comment:
Did you delete this because it's unused?

Reply:
Yes, it was moved to the custom forms controller